integration/networking/windows_drivers_test.go

```diff
@@ -55,13 +53,25 @@ func TestWindowsNetworkDrivers(t *testing.T) {
 			netName := fmt.Sprintf("test-%s-%d", tc.driver, tcID)
 
 			// Create network with specified driver
-			netID := network.CreateNoError(ctx, t, c, netName,
-				network.WithDriver(tc.driver),
-			)
+			netResp, err := c.NetworkCreate(ctx, netName, client.NetworkCreateOptions{
+				Driver: tc.driver,
+			})
+			if err != nil {
+				// L2Bridge may fail if host network configuration is not available
+				if tc.driver == "l2bridge" {
+					errStr := strings.ToLower(err.Error())
+					if strings.Contains(errStr, "hnscall failed") ||
+						strings.Contains(errStr, "network does not have a subnet") ||
+						strings.Contains(errStr, "not supported") {
+						t.Skipf("Driver %s requires host network configuration: %v", tc.driver, err)
```

Comment:
... mostly because `hnscall failed` may be a quite generic error; considering that the same as the (possibly expected) error could easily hide actual issues.